internal/processor/otel/metrics.go

```diff
@@ -176,6 +176,34 @@ func (b *apmMetricsBuilder) accumulate(m pmetric.Metric) {
 							b.cpuCount = cpuID + 1
 						}
 					}
+				case "process.runtime.jvm.memory.usage",
+					"process.runtime.jvm.memory.init",
+					"process.runtime.jvm.memory.committed",
+					"process.runtime.jvm.memory.limit":
+
+					var key jvmMemoryKey
+					dp.Attributes().Range(func(k string, v pcommon.Value) bool {
+						switch k {
+						case "type":
+							key.area = v.AsString()
+						case "pool":
+							key.pool = v.AsString()
+						}
+						return true
+					})
+
+					switch m.Name()[strings.LastIndex(m.Name(), ".")+1:] {
+					case "limit":
+						key.jvmType = "max"
+					case "usage":
+						key.jvmType = "used"
+					case "committed":
+						key.jvmType = "committed"
+					}
+
+					if key.jvmType != "" {
+						b.jvmMemory[key] = apmMetricValue{dp.Timestamp().AsTime(), sample.Value}
+					}
 				case "runtime.jvm.memory.area":
```

Comment:
Could you please add a comment to mention that `runtime.jvm.*` metrics were renamed in the OTel Java SDK v1.13.0 (https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v1.13.0)? We should remove this code some time in the future.

Reply:
Good point! Added